sw/device/silicon_creator/mask_rom/primitive_bootstrap.c

```diff
@@ -185,6 +187,10 @@ static rom_error_t bootstrap_flash(void) {
 }
 
 rom_error_t primitive_bootstrap(void) {
+  // TODO(lowRISC/opentitan#10631): decide on watchdog strategy for bootstrap.
+  watchdog_disable();
```

Comment:
nit: We can probably re-enable the watchdog on success as primitive_bootstrap doesn't reset the device.

Reply:
This is pretty problematic actually. Since primitive_bootstrap is entered unconditionally the watchdog timer gets disabled here and then isn't currently re-enabled at all. I'll need to think about this a little bit.

Reply:
We will replace this code anyway but #10809 modifies this a little bit: watchdog is disable only if bootstrap is requested and re-initialized on return.